src/types/redux/globals.ts

```diff
@@ -0,0 +1,55 @@
+import * as constants from './constants'
+import type { CountryData, UrlsConfig } from '~types/commons'
+import type { EnterpriseCobranding } from '~types/enterprise'
+import type { DocumentTypes, PoaTypes } from '~types/steps'
+
+export type SmsPayload = {
+  number?: string
+  valid?: boolean
+}
+
+export type GlobalActions =
+  | { type: typeof constants.SET_ID_DOCUMENT_TYPE; payload: DocumentTypes }
+  | { type: typeof constants.SET_ID_ISSUING_COUNTRY; payload: CountryData }
+  | { type: typeof constants.RESET_ID_ISSUING_COUNTRY }
+  | { type: typeof constants.SET_POA_DOCUMENT_TYPE; payload: PoaTypes }
+  | { type: typeof constants.SET_ROOM_ID; payload: string }
+  | { type: typeof constants.SET_SOCKET; payload: SocketIOClient.Socket }
+  | { type: typeof constants.SET_MOBILE_NUMBER; payload: SmsPayload }
+  | { type: typeof constants.SET_CLIENT_SUCCESS; payload: boolean }
+  | { type: typeof constants.MOBILE_CONNECTED; payload: boolean }
+  | { type: typeof constants.ACCEPT_TERMS }
```

Comment:
We can keep them for now, because the main purpose of these TS conversion PRs are to provide a foundation so we can implement new components or convert the current components to TS without being stuck with some missing types

Reply:
Yeah, fair enough. It might also end up getting cleaned up with the BIPA work anyway.